python-checks/src/main/java/org/sonar/python/checks/ParsingErrorCheck.java

```diff
@@ -19,38 +19,30 @@
  */
 package org.sonar.python.checks;
 
-import com.sonar.sslr.api.Grammar;
 import com.sonar.sslr.api.RecognitionException;
-import java.io.PrintWriter;
-import java.io.StringWriter;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.sonar.check.Priority;
 import org.sonar.check.Rule;
-import org.sonar.squidbridge.AstScannerExceptionHandler;
+import org.sonar.python.PythonCheck;
+import org.sonar.python.PythonVisitorContext;
 import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
-import org.sonar.squidbridge.checks.SquidCheck;
 
 @Rule(
     key = ParsingErrorCheck.CHECK_KEY,
     priority = Priority.MAJOR,
     name = "Parser failure")
 @SqaleConstantRemediation("30min")
-public class ParsingErrorCheck extends SquidCheck<Grammar> implements AstScannerExceptionHandler {
-  private static final Logger LOG = LoggerFactory.getLogger(ParsingErrorCheck.class);
-  public static final String CHECK_KEY = "ParsingError";
+// TODO integration test
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)